### PR TITLE
Feature/run ml pipeline

### PR DESCRIPTION
**Pull request by dsmits**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
 .idea
 venv/
 pgdata/
-__pycache__/
\ No newline at end of file
+__pycache__/.ipynb_checkpoints
```
Comment (svenvanderburg) [inline .gitignore]: What does this change exactly? This means anything in the `__pycache__` folder apart from `.ipynb_checkpoints` is added to git right? That's not what you want I guess? Or I forgot how `.gitignore` works.

File: test_federated_learning.ipynb [hunk 2]
```diff
@@ -0,0 +1,255 @@
+{
```
Comment (svenvanderburg) [inline test_federated_learning.ipynb:1]: I am used to having a rationale/TLDR on top of notebooks so if you ever go back to it you still know what the purpose of it was and what it does. Would be nice to add that here?

**Review by svenvanderburg — CHANGES_REQUESTED** [review]
See comments

File: .gitignore (hunk 1 above)
Comment (dsmits) [reply .gitignore]: Yeah you're right. I'll change it to ignoring everything in pycache

File: test_federated_learning.ipynb (hunk 2 above)
Comment (dsmits) [reply test_federated_learning.ipynb:1]: Added it

**Pull request merged**